utils/staking-miner/src/main.rs

```diff
@@ -276,15 +276,15 @@ struct DryRunConfig {
 #[derive(Debug, Clone, StructOpt)]
 struct SharedConfig {
 	/// The `ws` node to connect to.
-	#[structopt(long, short, default_value = DEFAULT_URI)]
+	#[structopt(long, short, default_value = DEFAULT_URI, env = "URI")]
```

Comment:
```suggestion
	#[structopt(long, default_value = DEFAULT_URI, env = "URI")]
```

Reply:
I double checked and this works fine.

It allows users to use `-u <uri>` (short) as well as `--uri <uri>` (long). The benefit is however admitedly very limited and saves only a little typing.

If there are strong opinions against it, I don't mind removing those. There are not many flags and no conflicts/ambiguities so I think that leaving `short` is fine.